react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -306,37 +459,113 @@ class Filmstrip extends PureComponent <Props> {
             toolbar = this._renderToggleButton();
         }
 
+        const filmstrip = (<>
+            <div
+                className = { clsx(this.props._videosClassName,
+                    !tileViewActive && !_resizableFilmstrip && 'filmstrip-hover') }
+                id = 'remoteVideos'>
+                {!_disableSelfView && !_verticalViewGrid && (
+                    <div
+                        className = 'filmstrip__videos'
+                        id = 'filmstripLocalVideo'>
+                        <div id = 'filmstripLocalVideoThumbnail'>
+                            {
+                                !tileViewActive && <Thumbnail
+                                    key = 'local' />
+                            }
+                        </div>
+                    </div>
+                )}
+                {
+                    this._renderRemoteParticipants()
+                }
+            </div>
+            <AudioTracksContainer />
```

Comment:
Side note: this should probably not be here. This could be a thing at the same level as dialog container or overlays since it's kinda "global".

Reply:
I'll move it in the parent div, but keep it in this component